src/f5_tts/model/cfm.py

```diff
@@ -98,17 +98,15 @@ def sample(
         edit_mask=None,
     ):
         self.eval()
-
-        if next(self.parameters()).dtype == torch.float16:
-            cond = cond.half()
-
         # raw wave
 
         if cond.ndim == 2:
             cond = self.mel_spec(cond)
             cond = cond.permute(0, 2, 1)
             assert cond.shape[-1] == self.num_channels
 
+        cond = cond.to(next(self.parameters()).dtype)
```

Comment:
actually if set this before the previous code block, it will cause bug.
before self.mel_spec, cond.dtype is float16
after self.mel_spec, cond.dtype is float32
so I move it after forward self.mel_spec